packages/flutter/lib/src/rendering/paragraph.dart

```diff
@@ -36,7 +36,7 @@ class RenderParagraph extends RenderBox {
   // TODO(abarth): This logic should live in TextPainter and be shared with RenderEditableLine.
   void _layoutText(BoxConstraints constraints) {
     assert(constraints != null);
-    assert(constraints.debugAssertIsNormalized);
+    assert(constraints.debugAssertIsValid());
```

Comment:
It can be if you think it should. I wasn't really sure what the semantics of minWidth for Paragraph were and wasn't sure if infinity made sense for that.